scripts/party.coffee

```diff
@@ -0,0 +1,55 @@
+# Description:
+#   Parties with two emoji
+#
+# Dependencies:
+#    none
+#
+# Configuration:
+#   None
+#
+# Commands:
+#    hubot party <emoji> <emoji> - ralph parties with your two emoji
+#    hubot party <emoji> <emoji> WxH - ralph parties with your two emoji in a W by H rectangle.
+#
+# Notes:
+#    None
+#
+# Author:
+#    @km42levelup
+
+DEFAULT_WIDTH = 12
+
+DEFAULT_HEIGHT = 6
+
+party = (emoji_one, emoji_two, width, height) ->
+    line_one = Array(Math.floor(width / 2) + 1).join(emoji_one + emoji_two)
+    if ((width % 2) == 1)
+        line_one += emoji_one
+
+    line_two = Array(Math.floor(width / 2) + 1).join(emoji_two + emoji_one)
+    if ((width % 2) == 1)
+        line_two += emoji_two
+
+    paragraph = ''
+    i = 0
+    while i < height
+        if ((i % 2) == 0)
+            paragraph += line_one + '\n\n'
```

Comment:
Are the double newlines needed here?

Reply:
Heres an example of the difference

![image](https://user-images.githubusercontent.com/41017188/67774289-77b2e680-fa33-11e9-81b2-f7bf7a3d74a5.png)

I think increasing density with just one newline is probably better so let's do that

Reply:
Ahh, gotcha. Yeah, all mine go full density.

Reply:
it is done